python/cm/stack.py

```diff
@@ -516,6 +517,27 @@ def get_yspec(proto, ref, bundle_hash, conf, name, subname):
     return schema
 
 
+def check_unique_names_constraint(conf_dict):
+    names = defaultdict(list)
+    firs_level_key = 'firs_level_key'
```

Comment:
Typo: first_level_key.
maybe value to 'root'?

Reply:
done